inception-app-webapp/src/main/java/de/tudarmstadt/ukp/inception/app/menu/MonitoringPageMenuItem.java

```diff
@@ -29,7 +29,7 @@
 import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
 import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
 import de.tudarmstadt.ukp.clarin.webanno.ui.core.menu.MenuItem;
-import de.tudarmstadt.ukp.clarin.webanno.ui.monitoring.page.MonitoringPage;
+import de.tudarmstadt.ukp.inception.app.ui.monitoring.page.MonitoringPage;
```

Comment:
Instead of changing the existing MonitoringPageMenuItem, please add a new WorkloadManagementPageMenuItem - I would like to be able to access both at the moment.